components/landingPage/About.tsx

```diff
@@ -2,21 +2,24 @@ import Image from "next/image";
 import Person from "./Person";
 
 export default function About() {
-	/**
-	 * The "About" Section In The Landing Page
-	 */
+	const data = [
+		{ fullName: "Name Surname 1", role: "role(s)", url: "https://www.linkedin.com", path: "/pics/face.png" },
+		{ fullName: "Name Surname 2", role: "role(s)", url: "https://www.linkedin.com", path: "/pics/face.png" },
+		{ fullName: "Name Surname 3", role: "role(s)", url: "https://www.linkedin.com", path: "/pics/face.png" },
+		{ fullName: "Name Surname 4", role: "role(s)", url: "https://www.linkedin.com", path: "/pics/face.png" },
+		{ fullName: "Name Surname 5", role: "role(s)", url: "https://www.linkedin.com", path: "/pics/face.png" },
+		{ fullName: "Name Surname 6", role: "role(s)", url: "https://www.linkedin.com", path: "/pics/face.png" },
+		{ fullName: "Name Surname 7", role: "role(s)", url: "https://www.linkedin.com", path: "/pics/face.png" }
```

Comment:
This looks smelly, redundant code.

Reply:
I know, our names should be there. Do you think it's time to put them there?

Reply:
Why not, it's not too much work and it would look much nicer. ;)

Reply:
We'll need pics, roles, and the LinkedIn URLs then, Should I ask for them on discord?

Reply:
Yap, good idea ;)